### ISSUE TITLE
False positive with Style/RedundantParantheses

### ISSUE THREAD

**Issue by eneagoe** [r7kamura/slimcop#18]
It looks like we have a false positive with this cop:

```ruby
app/views/layouts/_navigation.slim:34:25: C: [Correctable] Style/RedundantParentheses: Don't use parentheses around a logical expression.
```

The line in question is

```slim
    a.navbar-brand href=(current_customer&.logo_url.presence || root_path)
```

Without the parantheses we break the template, so they are definitely not redundant.

**Comment by r7kamura** [comment]
Thanks for reporting, you're absolutely right 👍
I think it would be reasonable to change the [default.yml](https://github.com/r7kamura/slimcop/blob/d1637860623a9037ceee5156e575b8103f375908/default.yml) to disable this cop, what do you think?

**Comment by eneagoe** [comment]
I agree - it would be very difficult to figure out when we can safely apply it.

**Comment by r7kamura** [comment]
Just released v0.19.0 👌 

- https://rubygems.org/gems/slimcop/versions/0.19.0
- https://github.com/r7kamura/slimcop/releases/tag/v0.19.0

**Comment by eneagoe** [comment]
Beautiful, thank you!

**Comment by eneagoe** [comment]
Actually there might be another issue - it looks like there's a typo somewhere - it should be `Style/RedundantParentheses` instead of `Style/RedundantParantheses`, sorry about that!

**Comment by r7kamura** [comment]
wow, I'll fix that soon 🤣
